specification.md

```diff
@@ -2268,8 +2267,9 @@ instance in case it is an end state without performing any actions.
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
 | name | Unique action name | string | no |
-| [functionRef](#FunctionRef-Definition) | References a reusable function definition | object | yes if `eventRef` is not used |
-| [eventRef](#EventRef-Definition) | References a `trigger` and `result` reusable event definitions | object | yes if `functionRef` is not used |
+| [functionRef](#FunctionRef-Definition) | References a reusable function definition | object | yes if `eventRef` & `subFlow` are not defined |
+| [eventRef](#EventRef-Definition) | References a `trigger` and `result` reusable event definitions | object | yes if `functionRef` & `subFlow` are not defined |
+| subFlow | Either string containing the unique id of the sub-workflow to be invoked or a [subFlowRef](#SubFlowRef-Definition) object | object or string | yes if `eventRef` & `functionRef` are not defined |
```

Comment:
Should we not call this "subflowRef" instead so it fits nicely with "eventRef and "functionRef"?

Reply:
For functions and events, we have other objects in the same document that the functionRef/eventRef are referring to.  For subflow, it's a completely different doc so it seemed odd to me to call it subFlowRef.  It's a minor thing, I'll defer to your judgement and rename it subFlowRef.